extractor/logInterpreter.py

```diff
@@ -144,11 +144,40 @@ def createEmptySessionDict(self, sessionID, time, sipTo, sipFrom):
             "messages": []
         }
     
+
     def checkIfDictKeysContainsRelatedSessions(self, sipHeader):
         pattern = re.compile(self.relatedSessionFormatPattern)
         matchingHeaders = [key for key in sipHeader if pattern.search(key)]
         return matchingHeaders
-    
+
+
+    def filterAssociatedSessions(self, dict, sessionID):
+        result = {}
+        relatedSessions = []
+
+        if sessionID == '':
+            print('No sessionID')
+            return json.dumps(list(dict.values()), indent=2)
+        else:
+            print('SessionID: ', sessionID)
+            # Finds relatedSessions for selected sessionID
+            if sessionID in dict:
+                for key in dict:
+                    if key == sessionID:

```

Comment:
Does only using dict[sessionID] instead of for key in dict and if key == sessionID work?